#23

jesspearson opened the pull request:
Added a registration journey.

User can only register if they have the 'Access code' which is known only to the blogsite owner.

Also updated the log in journey so that it now checks the users table in the DB instead of using constants that were previously hard coded.

jesspearson committed:
made registration page

jesspearson committed:
started register class

jesspearson committed:
registration functionality

jesspearson committed:
Updated login functionality

Login functionality is now controlled by the users table in the database

ClareMDavidson commented:
Looks great. Really swanky form!
Could we have a link to the registration page somewhere? Visible only when logged in, so that the Bloggers can invite people to join.
If possible, hash out the passwords.

jesspearson committed:
added registration link to nav

Added registration link to dropdown menu.

Also updated padlock button so it is correctly aligned, also now works on hover rather than click.

jesspearson committed:
changed text in dropdown

jesspearson committed:
Merge branch 'master' into #23
